kore/test/Test/Kore/Unparser.hs

```diff
@@ -216,21 +218,27 @@ test_parse =
         ]
 
 roundtrip ::
-    (HasCallStack, Unparse a, Eq a, Show a) => Gen a -> Parser a -> Property
+    (HasCallStack, Unparse a, Eq a, Show a) =>
+    Gen a ->
+    (FilePath -> Text -> Either String a) ->
```

Comment:
For compatibility with the old parser in this test, import your `Parser` newtype qualified and import the `Parser` type alias from `Kore.Parser.Parser`. Use that type synonym here, and qualify the reference to the `Parser` constructor below.

Reply:
I don't understand what you're suggesting here. Can you be more explicit? Specifically, you mention a Parser type alias in Kore.Parser.Parser, but no such type alias exists currently.